rholang/src/main/scala/coop/rchain/rholang/interpreter/normalize.scala

```diff
@@ -526,6 +530,15 @@ object ProcNormalizeMatcher {
         ProcVisitOutputs(input.par.prepend(foldedNew), bodyResult.knownFree)
       }
 
+      case b: PBundle =>
+        val targetResult = normalizeMatch(b.proc_, input.copy(par = VectorPar()))
+        if (targetResult.par.wildcard || targetResult.par.freeCount > 0) {
+          throw UnexpectedBundleContent(
+            s"Bundle's content shouldn't have free variables or wildcards.")
```

Comment:
Style issue: we shouldn't use s-strings unless we're actually substituting something in it.
knownFree contains all the line, column numbers where the free variables occurred.

Reply:
Fixed.